xbmc/addons/BinaryAddonCache.h

```diff
@@ -31,8 +31,9 @@ namespace ADDON {
 class CBinaryAddonCache : public Observer
 {
 public:
-  virtual ~CBinaryAddonCache();
+  virtual ~CBinaryAddonCache() { }
```

Comment:
I just noticed the segfault on Windows too, but this "empty body" destructor causes a compilation error on VS2013. (duplicate dtor body).
```error C2084: function 'ADDON::CBinaryAddonCache::~CBinaryAddonCache(void)' already has a body	C:\DATA\git\xbmc\xbmc\addons\BinaryAddonCache.cpp	29```